mava/systems/ff_ippo_rware.py

```diff
@@ -517,7 +517,7 @@ def run_experiment(_run: run.Run, _config: Dict, _log: SacredLogger) -> None:
     # Calculate total timesteps.
     n_devices = len(jax.devices())
     config["num_updates_per_eval"] = config["num_updates"] // config["num_evaluation"]
-    timesteps_per_training = (
+    steps_per_rollout = (
```

Comment:
It will be amazing if we can also print at the beginning of the training the expected total timesteps (as it is different depending on device num...)

Reply:
forget about this as it is solved on seperate config PR